webhooks-extension/test/util.sh

```diff
@@ -126,69 +126,103 @@ function install_tekton() {
 
 # Install Dashboard (https://github.com/tektoncd/dashboard)
 #
-# As of May 2nd this is a bit fraught since dashboard does not yet have images pushed to gcr.io
-# 
-# So: look to see if $GOPATH/src/github.com/tektoncd/dashboard exists
-# If it doesn't : check that the user is ok for us to clone and ko apply it
-# If it does: check that the user is ok for us to cd into and ko apply it
+# An option to clone dashboard is provided.
+# Installation process is either:
+# kubectl applying current release via github uri
+# kubectl applying gcr nightly build yaml
+# kubectl applying curled nightly build
+# ko apply development version
 
-function install_dashboard() { 
-  if [ -z "$1" ]; then
-        echo "Usage ERROR for function: install_dashboard [namespace]"
-        echo "Missing [namespace]"
-        exit 1
+function install_dashboard() {
+  if [ -z "$1" ] || [ -z "$2" ]; then
+      echo "Usage ERROR for function: install_webhooks_extension [namespace] [version] <sleepTime>"
+      [ -z "$1" ] && echo "Missing [namespace]"
+      [ -z "$2" ] && echo "Missing [version]"
+      exit 1
   fi
-  namespace="$1"
-  check GOPATH
-  check KO_DOCKER_REPO docker.io/your_docker_id
-
-  dashboard_dir=$GOPATH/src/github.com/tektoncd/dashboard 
-  if [ -d $dashboard_dir ]; then 
-    echo "Dashboard source detected at $dashboard_dir"
-    read -p "Do you wish to ko apply the current source? " -n 1 -r apply
+
+  version=$2
+  # Current Release
+  if [[ $version == "1" ]]; then
+    kubectl apply --filename https://github.com/tektoncd/dashboard/releases/download/v0.2.0/release.yaml
```

Comment:
Is there a *latest release* URL we could use instead so we needn't keep updating this when we do cut a new one?

Reply:
going to test with: `https://github.com/tektoncd/dashboard/releases/latest`